R/get_fscore.R

```diff
@@ -64,8 +65,9 @@ get_fs <- function(data, model = NULL, group = NULL,
                              alpha = est$alpha,
                              method = method,
                              fs_matrices = TRUE)
-    fscore_se <- sqrt(diag(attr(fscore, "av_efs")))
-    augment_fs(est, fscore, fscore_se)
+    # fscore_se <- sqrt(diag(attr(fscore, "av_efs")))
```

Comment:
Mark, do you want to clean up the codes that are commented out?

Reply:
I've dropped them now.